src/components/theme.d.ts

```diff
@@ -0,0 +1,9 @@
+import { ITheme as ChakraITheme } from '@chakra-ui/core';
```

Comment:
Will evolve with the refactoring, will then be into the `ThemeProvider` folder (or `theme` one?)